### PR TITLE
chore(middleware): Test error middleware.

### PR DESCRIPTION
**Pull request by jdetle**
Work brought to you courtesy PLOTCON.

### CODE REVIEW

File: test/renderer/epics/github-publish-spec.js [hunk 1]
```diff
@@ -108,18 +107,12 @@ describe('notifyUser', () => {
...
       position: 'tr',
       level: 'success',
-      action: {
-        label: 'Open Gist',
-        callback: function openGist() {
-          shell.openExternal(`https://nbviewer.jupyter.org/gistID`);
-        },
-      },
-    }))
+    });
```
Comment (rgbkrk) [inline test/renderer/epics/github-publish-spec.js:116]: 👍

File: test/renderer/middleware-spec.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+const expect = chai.expect
+import { errorMiddleware } from '../../src/notebook/middlewares';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+chai.use(sinonChai);
+
+describe('The error middleware', () => {
+
+  it('errors with a payload message when given', () => {
```
Comment (rgbkrk) [inline test/renderer/middleware-spec.js]: ...when given one (?)

**Review by rgbkrk — APPROVED** [review]

File: test/renderer/middleware-spec.js (hunk 2 above)
Comment (jdetle) [reply test/renderer/middleware-spec.js]: fixed

File: test/renderer/middleware-spec.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+const expect = chai.expect
+import { errorMiddleware } from '../../src/notebook/middlewares';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+chai.use(sinonChai);
+
+describe('The error middleware', () => {
+
+  it('errors with a payload message when given one', () => {
```
Comment (rgbkrk) [inline test/renderer/middleware-spec.js]: I realize now this wording probably came from other text I put in saying "X with message given". The when threw me off I think. I was about to merge, wasn't too big a deal to change.

File: test/renderer/middleware-spec.js (hunk 2 above)
Comment (rgbkrk) [reply test/renderer/middleware-spec.js]: I realize now this wording probably came from other text I put in saying "X with message given". The when threw me off I think. I was about to merge, wasn't too big a deal to change.

**Comment by jdetle** [comment]
¯_(ツ)_/¯ thanks for the feedback!

**Pull request merged**